### PR TITLE
COM207- Appointment Creation From

### PR DESCRIPTION
**Pull request by Rojanmhj2055**
Updated styling of appointment Creation form and added Page Not Found page.

### CODE REVIEW

File: angular-app/src/app/pages/admin-panel/admin-panel/admin-panel.component.css [hunk 1]
```diff
@@ -4,86 +4,6 @@ mat-toolbar{
   padding: 15px;
 }
 
-mat-tab-group{
```
Comment (rramsdany) [inline angular-app/src/app/pages/admin-panel/admin-panel/admin-panel.component.css:7]: Why are you deleting all this?

File: angular-app/src/app/pages/admin-panel/admin-panel/admin-panel.component.html [hunk 2]
```diff
@@ -1,7 +1,4 @@
-<head>
-  <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
-</head>
-
+<!-- Author:Ariel Camargo-->
```
Comment (rramsdany) [inline angular-app/src/app/pages/admin-panel/admin-panel/admin-panel.component.html:1]: This is introducing an old version of Admin Panel, you need to pull changes from main first before merging this PR

**Review by rramsdany — CHANGES_REQUESTED** [review]
I think you may have not pulled changes from main before starting this PR. Please pull from `origin main` before PR this because you're trying to push an old version of the `admin-panel` component. I see this branch is 13 commits behind so pulling will fix this.
 
Correct me if I'm wrong but I don't believe you did any new work for this component?

File: angular-app/src/app/page-not-found/page-not-found.component.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<mat-toolbar color="accent">
+  <mat-toolbar-row>
+    <p class="business-title">Chameleon <Br>Business Studio</p>
```
Comment (rramsdany) [inline angular-app/src/app/page-not-found/page-not-found.component.html:3]: Awesome page, we needed something like this!

File: angular-app/src/app/app-routing.module.ts [hunk 4]
```diff
@@ -12,20 +11,22 @@ import { SetupWizardComponent } from './pages/auth/setup-wizard/setup-wizard.com
 import { SignInFormComponent } from './pages/auth/sign-in-form/sign-in-form.component';
 import { SiteHeaderComponent } from './theme/site-header/site-header.component';
 import { SplashComponent } from './pages/dashboard/splash/splash.component';
+import { AppointmentCreateFormComponent } from './pages/appointments/appointment-create-form/appointment-create-form.component';
```
Comment (rramsdany) [inline angular-app/src/app/app-routing.module.ts:14]: Please keep the imports in this file in alphabetical order

File: angular-app/src/app/app.module.ts [hunk 5]
```diff
@@ -120,6 +122,7 @@ import { CalenderViewComponent } from './pages/appointments/appointment-calender
       { path: 'setup-wizard', component: SetupWizardComponent},
       { path: 'sign-in', component: SignInFormComponent },
       { path: 'site-header', component: SiteHeaderComponent },
+      {path:'**',component:PageNotFoundComponent}
```
Comment (Matt-Crow) [inline angular-app/src/app/app.module.ts:125]: Great idea, I like it!

**Review by Matt-Crow — APPROVED** [review]
Everything looks good to me except for the stuff @rramsdany pointed out. I'll approve it, but those changes he suggests should still be made

**Comment by rramsdany** [comment]
@Rojanmhj2055 You still need to correct the files from your first commit because this PR is aiming to revert the admin panel to a previous version

File: angular-app/src/app/app.module.ts [hunk 6]
```diff
@@ -120,6 +120,11 @@ import { CalenderViewComponent } from './pages/appointments/appointment-calender
       { path: 'setup-wizard', component: SetupWizardComponent},
       { path: 'sign-in', component: SignInFormComponent },
       { path: 'site-header', component: SiteHeaderComponent },
+<<<<<<< HEAD
```
Comment (rramsdany) [inline angular-app/src/app/app.module.ts:123]: You have to delete these boiler plate git code throughout these files.

File: angular-app/src/app/app.module.ts [hunk 7]
```diff
@@ -120,6 +120,11 @@ import { CalenderViewComponent } from './pages/appointments/appointment-calender
...
+      {path:'appointment-list',component:AppointmentListComponent},
+      {path:'**',component:PageNotFoundComponent}
+=======
+>>>>>>> parent of 637e1a1 (COM207- Appointment Creation From)
```
Comment (rramsdany) [inline angular-app/src/app/app.module.ts:127]: Here

File: angular-app/src/app/pages/appointments/appointment-create-form/appointment-create-form.component.css [hunk 8]
```diff
@@ -12,3 +12,34 @@
...
+
+.spinner{
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  min-width: 300px;
+}
+=======
+>>>>>>> parent of 637e1a1 (COM207- Appointment Creation From)
```
Comment (rramsdany) [inline angular-app/src/app/pages/appointments/appointment-create-form/appointment-create-form.component.css:45]: Here too

File: angular-app/src/app/pages/appointments/appointment-create-form/appointment-create-form.component.html [hunk 9]
```diff
@@ -6,10 +6,53 @@
     <mat-card-content>
         <form id="form" [formGroup]="appointmentForm" (ngSubmit)="submit()">
 
+<<<<<<< HEAD
```
Comment (rramsdany) [inline angular-app/src/app/pages/appointments/appointment-create-form/appointment-create-form.component.html:9]: Here

File: angular-app/src/app/pages/appointments/appointment-create-form/appointment-create-form.component.ts [hunk 10]
```diff
@@ -25,7 +25,11 @@ import { IAppointment } from '../../../models/interfaces/IAppointment';
 })
 export class AppointmentCreateFormComponent {
     appointmentForm = this.fb.group({
+<<<<<<< HEAD
+
+=======
```
Comment (rramsdany) [inline angular-app/src/app/pages/appointments/appointment-create-form/appointment-create-form.component.ts:30]: This file too

**Review by rramsdany — APPROVED** [review]

**Pull request merged**